api/src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 # Uncomment the following to turn on full SQL debugging
```

Comment:
Revert?

Reply:
Yeah, I'll remove the spaces.

Reply:
Oh, sorry, I misread the diff as changing what was commented out. I have no feelings about spaces versus no spaces here.

Reply:
I'll stick with spaces then, because that's what Atom automatically does with block comments here, and I see no strong reason to not follow that pattern.